### ISSUE TITLE
Extend the time it takes for items to despawn.

### ISSUE THREAD

**Issue by nosknut** [StargateMC/StargateMCPublic#382]
I managed to die in space, and dialed back 2 mins later. Everything was gone... I was on a ship... I thin the normal time should me 5 min?

**Comment by Bysokar** [comment]
There are no modifications to how items respawn on our server anymore.
Was the ship destroyed?

**Comment by nosknut** [comment]
No. It was completly intact. I died of suffacation cus the air gens were too slow and i was programming. I saw the items exploding from me when i died, and when i gated back it was all gone. I have checked and the vortex does not destroy items.

**Comment by Bysokar** [comment]
Well do some dropped item testing. See if things despawn and how long til they do.
